blue-xdma/src/Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: all clean
+
+BUILD_DIR ?= build
+
+BSC_FLAGS := -elab
+BSC_FLAGS += -verilog
+BSC_FLAGS += -p +:%/Libraries/AMBA_TLM3/Axi4
+BSC_FLAGS += -p +:%/Libraries/Bus
+BSC_FLAGS += -p +:%/Libraries/AMBA_TLM3/TLM3
+BSC_FLAGS += -p +:%/Libraries/AMBA_TLM3/Axi
+BSC_FLAGS += -p +:../BlueLib/src
+BSC_FLAGS += -p +:../BlueAXI/src
+BSC_FLAGS += -p +:./include
+BSC_FLAGS += -bdir $(BUILD_DIR)
+# BSC_FLAGS += -promote-warnings ALL
```

Comment:
should enable warning all

Reply:
The `BlueAXI` subproject brings some warnings during compilation. Therefore, enabling this option here would result in the entire project being unable to compile. The recommended approach is to compile the entire project first and then manually enable this option.